test/e2e/proxy.go

```diff
@@ -65,7 +65,10 @@ func proxyContext(version string) {
 	It("should proxy logs on node using proxy subresource [Conformance]", func() { nodeProxyTest(f, prefix+"/nodes/", "/proxy/logs/") })
 	It("should proxy to cadvisor using proxy subresource [Conformance]", func() { nodeProxyTest(f, prefix+"/nodes/", ":4194/proxy/containers/") })
 
+	// using the porter image to serve content, access the content
+	// (of multiple pods?) from multiple (endpoints/services?)
```

Comment:
it just starts a pod that serves content (determined by envvars in the podspec) over tls/non-tls ports